doc/DOCUMENTATION.org

```diff
@@ -3389,54 +3389,77 @@ Examples:
 ***** Lisp state key bindings
 These commands automatically switch to =lisp state=.
 
+****** Navigating
+
+| Key binding | Description                                                             |
```

Comment:
most of time it's easier to read if bindings are listed in lexicographical order.
but for directional keys such asj, h, k, l, it's clearer if they're grouped together.

also, pairs like `0` and `$` can be grouped together.

so

```org
| Key binding | Description                                                             |
|-------------+-------------------------------------------------------------------------|
| ~SPC k j~   | go to next closing parenthesis                                          |
| ~SPC k k~   | go to previous opening parenthesis                                      |
| ~SPC k l~   | go to next symbol                                                       |
| ~SPC k h~   | go to previous symbol                                                   |
| ~SPC k L~   | go to next sexp                                                         |
| ~SPC k H~   | go to previous sexp                                                     |
| ~SPC k U~   | go "up": to parent sexp backward                                        |
| ~SPC k 0~   | go back to the beginning of current sexp                                |
| ~SPC k $~   | go to the end of current sexp                                           |
| ~SPC k %~   | evil jump item                                                          |
| ~SPC k I~   | go back to beginning of current expression and switch to =insert state= |
```